lib/srv/regular/sshserver.go

```diff
@@ -973,7 +966,7 @@ func (s *Server) HandleNewConn(ctx context.Context, ccx *sshutils.ConnectionCont
 		return ctx, nil
 	}
 
-	netConfig, err := s.authService.GetClusterNetworkingConfig(ctx)
+	netConfig, err := s.GetAccessPoint().GetClusterNetworkingConfig(ctx)
```

Comment:
Any reason for this change?

Reply:
Although the change isn't necessary strictly speaking I think it's sensible for two reasons: 
1. To make it consistent with `s.GetAccessPoint().GetAuthPreference(ctx)` that's being added above in this same function.
2. To make the fact the config should be retrieved from a cached access point more explicit.

Reply:
Fair enough. I don't usually like shifting line blame unless there is a need or a compelling argument, that's why I asked.